### PR TITLE
Fn 1 header

### PR DESCRIPTION
**Pull request by JiriLaudat**
One line is out

### CODE REVIEW

**Review by fidelman — APPROVED** [review]

File: src/styles/components/header-nav.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.header-nav {
+
+  &__link {
+    color: theme-color('font-color');
+
+    &:active, &:hover {
```
Comment (fidelman) [inline src/styles/components/header-nav.scss]: Start selectors from a new line

File: src/styles/components/header-nav.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      color: theme-color('font-hover-color');
+      text-decoration: none;
+    }
+  }
+
+  &__icon {
+    width: rem(20px);
```
Comment (fidelman) [inline src/styles/components/header-nav.scss]: use size()

File: src/tpl/components/main-header/header-nav.nunj [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<ul class="header-nav list-unstyled d-flex m-0">
```
Comment (fidelman) [inline src/tpl/components/main-header/header-nav.nunj:1]: check if `list-unstyled` has maring: 0

File: src/tpl/components/main-header/index.nunj [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<header class="main-header fixed-top">
+	<div class="container d-flex">
```
Comment (fidelman) [inline src/tpl/components/main-header/index.nunj]: ccheck nesting

**Review by fidelman — CHANGES_REQUESTED** [review]

File: src/tpl/components/main-header/index.nunj (hunk 4 above)
Comment (JiriLaudat) [reply src/tpl/components/main-header/index.nunj]: Nesting seems to be OK. Two spaces are there

File: src/tpl/components/main-header/header-nav.nunj (hunk 3 above)
Comment (JiriLaudat) [reply src/tpl/components/main-header/header-nav.nunj:1]: m-0 is needed in this case, it makes mess if I delete it

**Review by fidelman — APPROVED** [review]

**Pull request merged**